Tests/SwiftStdlibTests/DictionaryExtensionsTests.swift

```diff
@@ -163,4 +163,20 @@ final class DictionaryExtensionsTests: XCTestCase {
             [6: ["Bryant"], 5: ["James"], 4: ["Wade", "John"], 0: ["", ""]])
         XCTAssertEqual(Dictionary(grouping: array3, by: \String.count), [:])
     }
+    
+    func testGetByKeys() {
+        let dict = ["James": 100,
+                    "Wade": 200,
+                    "Bryant": 500,
+                    "John": 600,
+                    "Jack": 1000]
+        
+        let res = dict[withKeys: "James", "Wade", "Jack"]
+        XCTAssertEqual(res, ["James": 100, "Wade": 200, "Jack": 1000])
+        
+        
```

Comment:
You have too many white spaces here. Can you please run `swiftformat`? There's a pre-commit hook you can install in `/scripts/pre-commit`.